src/routes/sign/Sign.svelte

```diff
@@ -43,9 +47,9 @@
 <h1>Sign</h1>
 
 {#if $xdr}
-    <Transaction txParams="{{ xdr: $xdr, description: $description }}" />
+    <Transaction txParams="{{ xdr: $xdr, description: $description, operationsGroups: $operationsGroups }}" />
 {:else if $isXdrNull}
-    <h1>Sorry, an XDR wasn't provided</h1>
+    <h1 class="test-cypress">Sorry, an XDR wasn't provided</h1>
```

Comment:
you should be using data- attributes https://docs.cypress.io/guides/references/best-practices#Selecting-Elements